# Размер пакета прошивки
int32 packet_size

# Пакет прошивки
int8[] firmware